### PR TITLE
Reproduce timeout bug in consumer

### PR DESCRIPTION
**Pull request by nicholastmosher**

### CODE REVIEW

File: tests/runner/src/test_runner.rs [hunk 1]
```diff
@@ -27,7 +27,8 @@ impl TestRunner {
             command
                 .arg("topic")
                 .arg("create")
-                .arg(&topic_name)
+                .arg("test-bug")
```
Comment (sehz) [inline tests/runner/src/test_runner.rs:30]: changed to something more meaningful `test-concurrent`

File: tests/runner/src/tests/smoke/concurrent.rs [hunk 2]
```diff
@@ -0,0 +1,68 @@
+use fluvio::{Fluvio, Offset};
+use futures_lite::StreamExt;
+use fluvio_future::task::spawn;
+use md5::Digest;
+use std::sync::mpsc::{Receiver, Sender};
+
+const TOPIC: &str = "test-bug";
```
Comment (sehz) [inline tests/runner/src/tests/smoke/concurrent.rs:7]: use the topic name int the test runner

File: tests/runner/src/tests/smoke/mod.rs [hunk 3]
```diff
@@ -24,8 +26,10 @@ mod runner {
         }
 
         async fn produce_and_consume_cli(&self) {
-            let start_offsets = super::produce::produce_message(&self.option).await;
```
Comment (sehz) [inline tests/runner/src/tests/smoke/mod.rs:27]: kick off concurrent test depends on test flag

**Review by sehz — CHANGES_REQUESTED** [review]

**Comment by nicholastmosher** [comment]
Superseded by new test runner

**Pull request closed without merging**